client/status/container.go

```diff
@@ -5,6 +5,13 @@ import (
 	"github.com/nspcc-dev/neofs-api-go/v2/status"
 )
 
+var (
+	// ErrEACLNotFound must not be mutated and should be used for errors.Is calls only.
```

Comment:
XXX is an instance of YYY error status. It's expected to be used for [errors.Is] and MUST NOT be changed.

Or is it obvious that we have an instance of something here?

Reply:
I my opinion such detailed comment about our expectations with errors.Is only, maybe too explicit. It's obvious, because it is a standard Error declaration. But to avoid any misunderstanding, replaced it